identity-service/src/captchaMiddleware.js

```diff
@@ -7,12 +7,14 @@ const verifyAndRecordCaptcha = async ({ token, walletAddress, url, logger, captc
     try {
       ({ score, ok, hostname } = await captcha.verify(token))
 
-      models.BotScores.create({
-        walletAddress,
-        recaptchaScore: score,
-        recaptchaContext: url,
-        recaptchaHostname: hostname
-      })
+      if ((score || (score === 0)) && hostname) {
```

Comment:
better to check
if (score !== undefined && score !== null) ?

Reply:
gotta love js

Reply:
https://www.destroyallsoftware.com/talks/wat